docker-compose.yml

```diff
@@ -9,9 +9,13 @@ services:
     build: ./
     env_file:
       - .env
+    environment:
+      MEMORY_BACKEND: ${MEMORY_BACKEND:-redis}
+      REDIS_HOST: ${REDIS_HOST:-redis}
     volumes:
-      - "./autogpt:/app"
-      - ".env:/app/.env"
+      - ./logs:/app/logs
```

Comment:
This is resolved in the orther PR I mentioned. We just copy and mount the entire project so the dir structure is completely consistent with running it outside docker